lib/fog/kubevirt/requests/compute/get_template.rb

```diff
@@ -3,7 +3,7 @@ module Compute
     class Kubevirt
       class Real
         def get_template(name)
-          Template.parse object_to_hash( client.get_template(name, 'default') )
+          Template.parse object_to_hash( oc_client.get_template(name, 'default') )
```

Comment:
Does the use of 'oc_client'  mean that this will work only on openshift ?

Reply:
Yes, this is exactly what we want.